catalog/core/catalog-core-app/src/main/resources/features.xml

```diff
@@ -0,0 +1,66 @@
+<!--
```

Comment:
This file should be removed and all features should be maintained in `catalog-app`.